### PR TITLE
cmd/action: Parse schema name in postgres dsn

### PR DESCRIPTION
**Pull request by CGA1123**
`lib/pq` supports parsing runtime parameters from the DSN. Allowing for
configuration of the schema to inspect or apply on using the
`search_path` parameter.

Closes: #457

### CODE REVIEW

**Review by a8m — APPROVED** [review]
Looks good to me. Thanks for the contribution @CGA1123 🙏

File: cmd/action/mux.go [hunk 1]
```diff
@@ -90,14 +91,34 @@ func SchemaNameFromDSN(url string) (string, error) {
...
+	// lib/pq supports setting default schemas via the `search_path` parameter
+	// in a dsn.
+	//
+	// See: https://github.com/lib/pq/blob/8446d16b8935fdf2b5c0fe333538ac395e3e1e4b/conn.go#L1155-L1165
+	queries := url.Query()
+	if queries.Has("search_path") {
+		return queries.Get("search_path"), nil
+	}
+
+	return "public", nil
```
Comment (a8m) [inline cmd/action/mux.go:119]: One comment, maybe skip it if the "search_path" is empty or return an error? (e.g. `"postgres://localhost:5432/dbname?search_path="`).

```suggestion
	url, err := url.Parse(dsn)
	if err != nil {
		// For backwards compatibility, we default to "public" when failing to
		// parse.
		return "public", nil
	}
	// lib/pq supports setting default schemas via the `search_path` parameter
	// in a dsn.
	//
	// See: https://github.com/lib/pq/blob/8446d16b8935fdf2b5c0fe333538ac395e3e1e4b/conn.go#L1155-L1165
	if schema := url.Query().Get("search_path"); schema != "" {
		return schema, nil
	}
```

Comment (a8m) [reply cmd/action/mux.go:119]: I approved the PR but then thought about the "empty value" case. 
cc @CGA1123 @masseelch

**Review by rotemtam — APPROVED** [review]
Thanks for your contribution @CGA1123 , if you haven't yet, please join our community on our discord server!

https://discord.com/invite/QhsmBAWzrC

**Pull request merged**